.github/actions/GetOfficialBranches/action.yaml

```diff
@@ -0,0 +1,29 @@
+name: Get Official Branches
+author: Microsoft Corporation
+description: Get the official branches in a repository
+inputs:
+  repository:
+    description: The repository to get the official branches from
+    required: true
+    default: ${{ github.repository }}
+outputs:
+  OfficialBranches:
+    description: The official branches in the repository
+    value:  ${{ steps.GetBranches.outputs.OfficialBranches }}
+runs:
+  using: composite
+  steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+
+    - name: run
+      shell: pwsh
+      id: GetBranches
+      env:
+        GH_TOKEN: ${{ github.token }}
```

Comment:
Does it need a github token? 🤔 Don't we usually just pass this when we use "gh cli"?

Reply:
Oh, I think that might be a leftover from another approach for getting the protected branches via the GitHub CLI.

Good catch!